Update Personal Finance (#374)
BitcoinDesign/Guide: guide/designing-products/personal-finance.md

# Personal finance
{:.no_toc}

**If we consider Bitcoin a new form of money, it’s only natural that we compare it to existing forms of currency.This section views Bitcoin through the lens of traditional finance to help us build mental models, define behavioral patterns, and ultimately better understand what kind of products we should be making.**

> This section views bitcoin through the lens of traditional finance to help us build mental models, define behavioral patterns, and ultimately better understand what kind of products we should be making. Because Bitcoin is both different and new, we can’t depend on solutions to it that worked for conventional financial products.

Reviewer: This paragraph (starting with "This section") is now duplicated. Could you please double-check and ensure the correct text is being added and removed here and in the surrounding paragraphs?

Author: oops nice catch fixed https://github.com/BitcoinDesign/Guide/pull/374/commits/a9be74dce4bc3cbf5be164e4504eb70b01ca63d5

# Personal finance
{:.no_toc}

> **If we consider Bitcoin a new form of money, it’s only natural that we compare it to existing forms of currency.This section views Bitcoin through the lens of traditional finance to help us build mental models, define behavioral patterns, and ultimately better understand what kind of products we should be making.**

Reviewer 2: ```suggestion
**If we consider Bitcoin a new form of money, it’s only natural that we compare it to existing forms of currency. This section views Bitcoin through the lens of traditional finance to help us build mental models, define behavioral patterns, and ultimately better understand what kind of products we should be making.**
```

**If we consider Bitcoin a new form of money, it’s only natural that we compare it to existing forms of currency.This section views Bitcoin through the lens of traditional finance to help us build mental models, define behavioral patterns, and ultimately better understand what kind of products we should be making.**

> Money is complex and can play many different roles in our lives. Buying a new shirt is quite different from buying a house or setting aside savings for retirement. Taking a payment as a store owner is very different from a child receiving an allowance from parents. On a global scale, the differences multiply across countries and cultures.

Author: @ConorOkus What do you think?

> Money is complex and can play many different roles in our lives. Buying a new shirt is quite different from buying a house or setting aside savings for retirement. Taking a payment as a store owner is very different from a child receiving an allowance from parents. On a global scale, the differences multiply across countries and cultures.

Reviewer 2: It's useful information and helps to frame things, I would keep it

> Money is complex and can play many different roles in our lives. Buying a new shirt is quite different from buying a house or setting aside savings for retirement. Taking a payment as a store owner is very different from a child receiving an allowance from parents. On a global scale, the differences multiply across countries and cultures.

Author: @GBKS @ConorOkus  So how do we proceed from here should I revert the removal of all three paragraphs or just the one for which the comment is aimed at?

> Money is complex and can play many different roles in our lives. Buying a new shirt is quite different from buying a house or setting aside savings for retirement. Taking a payment as a store owner is very different from a child receiving an allowance from parents. On a global scale, the differences multiply across countries and cultures.

Reviewer: My comment was meant for all three paragraphs.

> Money is complex and can play many different roles in our lives. Buying a new shirt is quite different from buying a house or setting aside savings for retirement. Taking a payment as a store owner is very different from a child receiving an allowance from parents. On a global scale, the differences multiply across countries and cultures.

Reviewer 2: Yep, I was thinking we should keep all three also.

> Money is complex and can play many different roles in our lives. Buying a new shirt is quite different from buying a house or setting aside savings for retirement. Taking a payment as a store owner is very different from a child receiving an allowance from parents. On a global scale, the differences multiply across countries and cultures.

Author: @GBKS @ConorOkus 
Reverted the removal of the three paragraphs in question in https://github.com/BitcoinDesign/Guide/pull/374/commits/f21638651fab8ff71aefdd5755ef1de8fc932240

   layout = "float-left"
%}

> This is the “cash”—or in this case of Bitcoin, the “sats”—that we use to make small purchases such as coffee, lunch, etc. While some countries fully embrace credit cards and smartphones for daily spending, others, such as Germany, still prefer physical coins and bills.

Reviewer 3: I would keep a reference to 'why' people in Germany likes cash as it's relevant.
'..., such as Germany, still prefer cash for it's better privacy.'  or similar.

Reviewer: ```suggestion
This is the “cash”—or in this case of Bitcoin, the “sats”—that we use to make small purchases such as coffee, lunch, etc. While some countries fully embrace credit cards and smartphones for daily spending, others, such as Germany, still prefer cash for its better privacy.
```

   height = 800
%}

Most stores (online and offline) don’t currently accept bitcoin. When they do, users who already pay with smartphones or NFC-enabled credit cards can easily transition to Bitcoin apps.

> A typical scenario could be that users create dedicated mobile bitcoin (and lightning) wallets for on-the-go payments, in addition to having separate wallets for larger amounts. Similar to having a bank account and regularly taking out cash at an ATM, users could refill their mobile wallets. The mobile wallet may use [automatic cloud backup private key storage]({{ '/guide/private-key-management/cloud-backup/' | relative_url }}) The primary wallet however, would be more strongly secured with a [hardware wallet]({% link guide/getting-started/hardware.md %}#hardware-wallets) or even a [multi-key]({{ '/guide/private-key-management/multi-key/' | relative_url }}) configuration. This would mirror the primary focus of convenience over security. Beyond key management, payment interactions could be identical to what users are already familiar with.

Reviewer 2: ```suggestion
A typical scenario could be that users create dedicated mobile bitcoin (and lightning) wallets for on-the-go payments, in addition to having separate wallets for larger amounts. Similar to having a bank account and regularly taking out cash at an ATM, users could refill their mobile wallets. The mobile wallet may use [automatic cloud backup private key storage]({{ '/guide/private-key-management/cloud-backup/' | relative_url }}). The primary wallet however, would be more strongly secured with a [hardware wallet]({% link guide/getting-started/hardware.md %}#hardware-wallets) or even a [multi-key]({{ '/guide/private-key-management/multi-key/' | relative_url }}) configuration. This would mirror the primary focus of convenience over security. Beyond key management, payment interactions could be identical to what users are already familiar with.
```

   layout = "float-left"
%}

> Higher value payments such as rent, utilities, car payments, and groceries are seldom paid for in cash. Instead, they are tied to service providers with ingrained payment mechanisms such as checks, debits, ACH, and wire transfers.

Reviewer 3: ```suggestion
Higher value payments such as rent, utilities, car payments, and groceries are seldom paid for in cash. Instead, they are tied to service providers with ingrained payment mechanisms such as checks or bank transfers.
```

Reviewer 3: More concise and less US centric.
(I don't know what 'a debit' is, I do know what ACH is but only because I have lived in the US)

</div>

For recurring payments, some of these mechanisms let businesses regularly deduct money from accounts without customers approving each transaction.

> Bitcoin is different since the sender must initiate all transactions, meaning that automated deductions might not be possible—at least when the wallet is non-custodial. Lightning wallets, which are always online, could solve this by allowing more complex interactions between senders and recipients.

Reviewer 3: ```suggestion
Bitcoin is different since the sender must initiate all transactions, meaning that automated deductions might not be possible—at least when the wallet is non-custodial.
```

Reviewer 3: Mentioning Lightning as a solution here, fleetingly, feels unnecessary.
We don't yet cover it in the guide and have no info on how to achieve this, yet.

   height = 800
%}

> A hardware wallet accompanied by a desktop application where keys are stored and transactions are securely signed by hardware is another solution. This reduces the risk of keeping funds on a mobile wallet configuration but adds acceptable friction for transactions that occur less frequently.

Reviewer: Does not make sense anymore after the changes. The previous paragraph talks about how this typically works when using a bank for these transactions. This paragraph talks about how you would do this with Bitcoin. So it's not `another solution`.

Author: Good catch, so one way in which I tried to fix it is by removing that reference in https://github.com/BitcoinDesign/Guide/pull/374/commits/a9be74dce4bc3cbf5be164e4504eb70b01ca63d5 another way would be to revert both of those parts, let me know which one you prefer.

   layout = "float-left"
%}

This is money that one sets aside for a considerable but planned expense, such as a new car or a down payment on a house. The amount might be similar to an emergency budget but on a shorter timeline. There are likely to be multiple deposits and potentially only a single withdrawal, after which the fund is no longer needed.

</div>

> Sinking funds do not necessarily require a dedicated account. The amount secured could be deposited into any monthly budgeting or savings wallet.

Reviewer: ```suggestion
Sinking funds do not necessarily require a dedicated wallet. The amount secured could be deposited into any monthly budgeting or savings wallet.
```
Are we using account and wallet correctly here? A sinking fund could be a second account in a wallet, assuming that your wallet allows for multiple accounts (which many applications don't).

   height = 800
%}

> Of course, many prefer that banks and other custodial financial service providers secure and invest funds for them. Savings accounts and related products may also incentivize customers with low-interest rates that may or may not be high enough to keep up with inflation.

Reviewer: ```suggestion
Of course, many prefer that banks and other custodial financial service providers secure and invest funds for them. Savings accounts and related products may also incentivize customers with low interest rates that may or may not be high enough to keep up with inflation.
```

   layout = "float-left"
%}

Investing is about turning the money you have into more money over time. From individuals to governments to corporations to pension funds, the goal of investing is growth. This necessarily requires the involvement of third parties, which is beyond the focus of this guide (at least for now).

</div>

> Since Bitcoin continues to increase in value over time, many see the act of “HODLing” as investing. In the last ten years, an increasing number of financial advisors and institutions have come to see bitcoin as a necessary component or any sufficiently diversified portfolio.

Reviewer 2: ```suggestion
Since Bitcoin continues to increase in value over time, many see the act of “HODLing” as investing. In the last ten years, an increasing number of financial advisors and institutions have come to see bitcoin as a necessary component of any sufficiently diversified portfolio.
```

   layout = "float-left"
%}

> Investing is about turning the money you have into more money over time. From individuals to governments to corporations to pension funds, the goal of investing is growth. This necessarily requires the involvement of third parties, which is beyond the focus of this guide (at least for now).

Reviewer 3: ```suggestion
Investing is about turning the money you have into more money over time. From individuals to governments to corporations to pension funds, the goal of investing is growth. This necessarily requires the involvement of third parties, which is beyond the focus of this guide.
```

Reviewer 3: It reads well, but I really don't think the Bitcoin Design Guide will cover investing.
